docs/_rtd/data_types.md

```diff
@@ -23,27 +23,55 @@ Shape: 2D matrix of cells x cell_features
-* `centroid-1`: the y-coordinate of the centroid
+* `channel_1` -> `channel_n`: The unique channels for each marker used. 
+* `label`: A unique numerical label for each cell (row in the cell table). 
+* `area`: area of the cell. 
+* `eccentricity`: eccentricity of the ellipse that has the same second-moments as the region. The eccentricity is the ratio of the focal distance (distance between focal points) over the major axis length. The value is in the interval [0, 1). When it is 0, the ellipse becomes a circle. 
+* `major_axis_length`: the length of the major axis of the ellipse that has the same normalized second central moments as the region. 
+* `minor_axis_length`: the length of the minor axis of the ellipse that has the same normalized second central moments as the region. 
+* `perimeter`: perimeter of object which approximates the contour as a line through the centers of border pixels using a 4-connectivity. 
+* `convex_area`: the area of the convex hull. 
+* `equivalent_diameter`: the diameter of the circle with the same area as the cell. 
+* `centroid-0`: the $x$-coordinate of the centroid. 
+* `centroid-1`: the $y$-coordinate of the centroid 
+* `fov`: The FOV from which the cell originates from. 
 
 The base `regionprops` metric often don't provide enough morphological information about each cell on their own. We add the following derived metrics to provide more complete information about the segmented cells:
-* `major_minor_axis_ratio`: the major axis length divided by the minor axis length
-* `perim_square_over_area`: the square of the perimeter divided by the area
-* `major_axis_equiv_diam_ratio`: the major axis length divided by the equivalent diameter
-* `convex_hull_resid`: the difference between the convex area and the area divided by the convex area
-* `centroid_dif`: the normalized euclidian distance between the cell centroid and the corresponding convex hull centroid
-* `num_concavities`: the number of concavities of the region
-* `nc_ratio`: for nuclear segmentation only. The nuclear area divided by the total area.
+* `major_minor_axis_ratio`: the major axis length divided by the minor axis length. 
+* `perim_square_over_area`: the square of the perimeter divided by the area. 
+* `major_axis_equiv_diam_ratio`: the major axis length divided by the equivalent diameter. 
+* `convex_hull_resid`: the difference between the convex area and the area divided by the convex area. 
+* `centroid_dif`: the normalized euclidian distance between the cell centroid and the corresponding convex hull centroid. 
+* `num_concavities`: the number of concavities of the region 
+* `nc_ratio`: for nuclear segmentation only. The nuclear area divided by the total area. 
+
+If you are using your own spatial analysis tools to generate information for the Cell Table, adjust the following values in `ark/.settings.py`:
+* `PRE_CHANNEL_COL`: From `CELL_SIZE` to the parameter which occurs before the channel names.
```

Comment:
This wording makes it sound like they should specify the range of values between cell_size and their value. I would update it to just tell that what value to place here